packages/terra-clinical-site/package.json

```diff
@@ -29,9 +29,11 @@
   "dependencies": {
     "prop-types": "^15.5.8",
     "react-intl": "^2.3.0",
+    "terra-alert": "^1.0.0",
```

Comment:
I need to investigate this some. For some reason if I don't include terra-alert as a direct dependency the translations in the base alert component aren't pulled in.